### ISSUE TITLE
Gutenberg doesn't see changes in Blocks' Javascript files

### ISSUE THREAD

**Issue by creativiii** [WordPress/gutenberg#23139]
I was planning to use `wp-env` to develop some blocks, however my blocks aren't changing after editing them.

I'm loading the plugins in `.wp-env.json`
```
	"plugins": [
		".",
		"../my-plugin"
	],
```

I can make changes to the PHP files that make up the plugin just fine and they will be shown on the local Wordpress installation, but even after building/deactivating/activating the plugin the changes to the Javascript part of the block will not be shown.

I've checked my built file multiple times and the file has all the changes I made, but Wordpress still refuses to show the modifications.

Is there a cache setting of some sort I'm not aware of? How do I get my block to change?

**Comment by creativiii** [comment]
The changes work correctly when starting `wp-env` from the plugin folder. I think this might just be me misunderstanding how to use `wp-env.json`.

**Comment by noahtallen** [comment]
Hi! That's weird. There are no caches in wp-env: everything listed in `plugins` is mounted directly as a Docker volume, which means that any change to the filesystem is always up-to-date.

You need to run `wp-env start` from the same directory as `.wp-env.json` for it to work, though. Could you paste the output of `wp-env start --debug`?

**Comment by noahtallen** [comment]
`wp-env start` basically looks at each `$CWD` differently. So if you run in directory A, it will use the configuration in that directory, or if you run in directory B, it will use the configuration in that directory. If no local configuration is found in $CWD, it will attempt to detect the type of source. So if you run `wp-env start` from a plugin directory which has no .wp-env.json, it will attempt to see if the source is a plugin/theme/wordpress, and then mount it automatically.

**Comment by creativiii** [comment]
Right so, I'm not entirely sure what happened. I tried getting this to work for another couple of days with no luck, so I ended up running `wp-env` in my plugin folder, which worked fine so I just went with it and finished developing it there.

I tried starting Wordpress from the Gutenberg folder again today and the changes are appearing. 

Issue is solved, although I'm not sure what caused it and how it fixed itself. Sorry!
